Misha/Files/zip_creator/zip_test.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+import os
+from zip_main import check_set_of_paths, create_archive
+import zipfile
+
+
+class ZipTest(unittest.TestCase):
+
+    def test_check_set_of_paths_with_wrong_paths(self):
+        set_of_paths = ['path/to/file.1', 'path.txt', 'file.1']
+        with self.assertRaises(FileNotFoundError) as context:
+            self.assertRaises(FileNotFoundError, check_set_of_paths(set_of_paths))
+        exception_message = str(context.exception)
+        self.assertEqual('List of paths is empty', exception_message)
+
+    def test_create_archive_with_right_set_of_paths(self):
+        set_of_paths = ['zip_main.py', 'zip_test.py']
```

Comment:
Please create new folder with name 'test-resources' add some files there and archive them. You can add them to this commit 
You can use any file (picture, other archive, text file etc.)
Do not forget to clean this dir after tests. Try to find some 'after' phase for tests in the doc